lib/User.php

```diff
@@ -229,4 +243,81 @@ public function path() {
 	public function slug() {
 		return $this->user_nicename;
 	}
+
+	/**
+	 * Create the array with user roles names and slugs.
+	 *
+	 * @api
+	 * @since 1.8.5
+	 *
+	 * @return array|null
+	 */
+	public function get_roles( $roles ) {
```

Comment:
Shouldn’t this method be protected? Roles should only be accessed through the `roles()` method. Instead of the `@api` tag, it would then need an `@internal` tag. And there’s also a `@param` tag missing.